charter.html

```diff
@@ -285,13 +296,18 @@ <h2 id=work-items>Work Items</h2>
 <p>The Chairs may add Work Items, but must not add Work Items which lack
 the support of at least two <a href=#implementer>implementers</a>.
 
-<p class=note>Typically, Work Items begin life as a Proposal before
-being adopted.
+<p>Each Work Item should be accompanied by an explainer describing its
+proposed changes to the web platform. Editors should keep the Work
+Item's explainer up-to-date with the Work Item as it evolves.
+
+<p class=note>Since Work Items typically begin life as a Proposal before
+being formally adopted, they usually start out with an explainer.
 
 <p>When a Work Item's Editors deem the Work Item ready for migration to
-a standards body, they will notify the Chairs. The Editors and Chairs
-will then work together to identify the best destination, and will work
-with the destination standards body to successfully migrate the work.
+a standards body, they will notify the Chairs. The CG may produce a
+Final Community Group Report at this time. The Editors and Chairs will
+identify the best destination standards body, and will then work with
+the destination standards body to successfully migrate the Work Item.
```

Comment:
Maybe this would be a good place to re-introduce the idea (removed in the edit above just under the `#work-items` header) that the end result might be modifications to existing specifications?  Or did you want to get rid of that?

Reply:
I've tried to do this in 2d7f111.